ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -249,6 +249,17 @@ let start_proxy ~__context vif proxy =
       | Api_errors.Server_error (code, args) when
           code = Api_errors.openvswitch_not_active ->
         "Host is not using openvswitch"
+      | Api_errors.Server_error ("SR_BACKEND_FAILURE_79", _) ->
```

Comment:
So this error makes more sense in this context, since the cache/VDI has a valid size but there just isn't enough space on the SR; shouldn't this be the error that is thrown in this case?